cmaa.sevsnp.attest.sample/AttestationClient.cs

```diff
@@ -0,0 +1,76 @@
+using Newtonsoft.Json;
+using System.Text;
+
+public class AttestationClient
```

Comment:
Should this code use the MAA .NET SDK? The attest API called in MAA doesn't have to match the attestation evidence type included in the MAA certificate.